Debug-custom-ESLint-check-tests.md

```diff
@@ -0,0 +1,101 @@
+## Table of contents
+
+* [Introduction](#introduction)
+* [Print statements](#print-statements)
+* [Inspect the AST](#inspect-the-ast)
+
+## Introduction
+
+Especially for complex lint rules, it can be hard to figure out why a test failed. This page will give you some tips to help you debug your failing tests.
+
+## Print statements
+
+If you add a `console.log()`, `console.error()`, or `console.info()` statement to your custom ESLint rule code, the output will appear along with the test status output. For example, suppose we have a lint rule that raises an error every time `.click()` or `.sendKeys()` is called on an object instead of `action.click()` or `action.sendKeys()`, which we would like people to use instead. If we add `console.log()` statement to print `DEBUGGING found lint error` whenever our lint rule raises an error, we see this output when running the tests:
```

Comment:
If we add --> If we add a

Reply:
I'd also recommend not starting with "If you add a ...". Have the "If" describe the scenario the developer is facing and then provide the "add a" as a suggestion of what to do in that scenario.

Otherwise everything seems to be founded on hypotheticals, which is confusing.